src/config.yml

```diff
@@ -123,6 +127,11 @@ blocked_commands:
   - 's:b:/lrbuild:_'
   - 's:b:/defaultgamemode:_'
   - 's:b:/reload:_'
+  # Soon to be blocked TFM Command i think!
+  - 's:b:/spectator:_' 
+  # Idk if i should put this here
```

Comment:
Really just /summon would make more sense to be blocked, as for the most part its just to summon entities that you could create otherwise and would work....